tests/unit/parser.test.ts

```diff
@@ -10,7 +10,7 @@ const example_body = 'children: \r\n- https://github.com/protocol/bedrock/issues
 /**
  * Test data obtained from calling getIssue() on github.com/ipfs/ipfs-gui/issues/106 on 2023-02-10 @ 5:52pm PST
  */
-const example_tasklist_body = 'eta: 2023Q4\r\n\r\nchildren:\r\n- [ ] #121\r\n- [ ] #122\r\n- [ ] #123\r\n- [ ] #124\r\n\r\n```[tasklist]\r\n### Tasks\r\n- [ ] #121\r\n- [ ] #122\r\n- [ ] #123\r\n- [ ] #124\r\n```\r\n'
+const example_tasklist_body = 'eta: 2023Q4\r\n\r\nchildren:\r\n- [ ] #121\r\n- [ ] #122\r\n- [ ] #123\r\n- [ ] #124\r\n\r\n```[tasklist]\r\n### Tasks\r\n- [ ] #125\r\n- [ ] #126\r\n- [ ] #127\r\n- [ ] #128\r\n```\r\n'
```

Comment:
the issues in "children" and "tasklist" here were the same, and they shouldn't be (so we can ensure the correct ones are being read).

This change and the update to the expectation below prove that we prioritize reading tasklists over children.

Reply:
FYI that I broke this test when fixing #360, which is why the change is in this PR.